packages/api-runner/lib/fetchers/getPools.js

```diff
@@ -530,19 +557,23 @@ const calculateTotalValueLockedPerPool = (pools) =>
  * @param {*} pools
  * @returns
  */
-const calculateTokenFaucetApr = (pools) =>
+const calculateTokenFaucetAprs = (pools) =>
```

Comment:
Should we add this to `tokens.tokenFaucetDripTokens` as well?

The more I think about it the more I think having them accessible from `tokenFaucets[0].dripToken` is enough and that we should just drop `tokens.tokenFaucetDripTokens`.

Reply:
1) No, as the APR can differ from faucet to faucet. It's a faucet-specific thing.

2) We need to have the drip token assets in the `tokens` list in order to get token price data. So we _could_ clear that key out from `tokens` after the fact, but I felt it made more sense to simply leave it there.